app/src/main/java/com/wpi/audiojournal/screen/DataLoadSplashScreen.kt

```diff
@@ -0,0 +1,36 @@
+package com.wpi.audiojournal.screen
+
+
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.CircularProgressIndicator
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.colorResource
+import androidx.compose.ui.unit.dp
+import com.wpi.audiojournal.ui.theme.ColorScheme
+import com.wpi.audiojournal.ui.theme.LocalColorScheme
+import com.wpi.audiojournal.ui.theme.RainbowColorScheme
+
+@Composable
+fun DataLoadSplashScreen() {
```

Comment:
#19 also implements a loading screen for data, as a higher-order component. We should be consistent in what approach we use.